samples/Actor/DemoActor/Program.cs

```diff
@@ -31,9 +32,17 @@ public static void Main(string[] args)
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
                 .UseStartup<Startup>()
-            .UseActors(actorRuntime =>
-            {
-                actorRuntime.RegisterActor<DemoActor>();
-            });
+                .UseActors(actorRuntime =>
+                {
+                    actorRuntime.RegisterActor<DemoActor>();
+
+                    // Optionally register custom serialization options for inbound Actor requests
+                    actorRuntime.ConfigureJsonSerializationOptions(options =>
+                    {
+                        options.PropertyNameCaseInsensitive = true;
+                        options.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
+                        //options.Converters.Add(new MyCustomConverter();
```

Comment:
Is there a reason why this needs to be a callback style API? It's clunky compared with just having a property.